src/sonic-config-engine/tests/sample_output/py2/buffers-dell6100.json

```diff
@@ -637,4 +637,16 @@
             "profile" : "[BUFFER_PROFILE|egress_lossy_profile]"
         }
     }
+,
+    "DEFAULT_LOSSLESS_BUFFER_PARAMETER": {
```

Comment:
>DEFAULT_LOSSLESS_BUFFER_PARAMETER [](start = 5, length = 33)

Is it possible to keep other vendor's file intact?

Reply:
ok. this needs to be done in `files/build_templates/buffers_config.j2`